.changeset/rude-cobras-accept.md

```diff
@@ -0,0 +1,14 @@
+---
+'sku': major
```

Comment:
```suggestion
'sku': minor
```
As [changes to formatting are considered non-breaking](https://seek-oss.github.io/sku/#/./docs/linting?id=linting-and-formatting) I would suggest we make this a `minor`